azure-spring-boot-parent/pom.xml

```diff
@@ -65,6 +66,26 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk9+</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>${java.version}</maven.compiler.release>
+                <javadocExecutable>${java.home}/bin/</javadocExecutable>
```

Comment:
The directory structure of JDK9+ changed and the javadoc executable has been relocated.